SocialConstants.js

```diff
@@ -17,4 +18,12 @@ const youtube = {
     glyphPath: youtubeIconAsset,
 }
 
-export const SOCIAL_PLATFORMS = [spotify, youtube]
+const apple = {
+    id: 'apple',
+    color: { primary: '#000' , secondary: '#fff' },
+    size: { width: 25, height: 25 },
+    friendlyName: 'Apple',
```

Comment:
Sorry one more change for ya KAPPA

Let's change our friendlyName here to be Apple Music as I believe it would make more sense to say "Sign In With Apple Music"